src/components/People.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      input: "",
+      name: "",
+      age: "",
+      gender: "",
+      errPing: "",
+    };
+  }
+  handleInputChange = (e) => {
+    this.setState({
+      input: e.target.value,
+    });
+    console.log(this.state.input);
+  };
+
+  handleClick = async () => {
+    try {
+      const { data } = await axios.get(
+        `https://ghibliapi.herokuapp.com/people`
+      );
+      console.log(data);
+
+      const uuid = data.find((person) => person.name === this.state.input);
```

Comment:
Nice use of .find()! I'd recommend a different variable name since the thing returned by .find() is not a uuid